src/handlers/PanGestureHandler.ts

```diff
@@ -41,8 +41,26 @@ export type PanGestureHandlerEventPayload = {
   velocityY: number;
 };
 
+interface CommonPanProperties {
+  minDist?: number;
+  avgTouches?: boolean;
+  enableTrackpadTwoFingerGesture?: boolean;
+}
```

Comment:
Why do we need to extract it? It looks like its used only in PanGestureHandlerProps

Reply:
It's also used in `PanGestureConfig`. In other handlers props are extending the config, but `PanGestureHandler` is the only one that makes use of `transformProps`. I copied the logic of that in c207736e230eaea9e6e0261b51b19e1c411c7214, and `PanGestureConfig` holds already transformed properties, while those in `PanGestureHandlerProps` are yet to be transformed, so I extracted the ones they shared to separate object.